src/components/pages/VaultDetail/index.tsx

```diff
@@ -62,6 +63,13 @@ export const VaultDetail = (): JSX.Element => {
 
   return (
     <div className="p-4 space-y-8">
+      <Helmet>
+        <title>DAO Books - vault details</title>
```

Comment:
If its treasury the title should be 

`Treasury | <Dao Name> | DAO Books`

if Its minion then the title should be 

`<Minion Name> | <DAO Name> | DAO Books`